### PR TITLE
Add new 5.0 for set, get, capture affinity

### PR DESCRIPTION
**Pull request by tmh97**
Tests support for omp_set_affinity_format, omp_get_affinity_format, and omp_capture_affinity.

Noticed an issue with displaying affinity using omp_get_affinity and OMPVV_INFOMSG (and printf). 
Same error with gcc and llvm
`line 40: printf: ` ': invalid format character`

### CODE REVIEW

File: tests/5.0/program_control/test_set_get_capture_omp_affinity.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+int main(void) {
+
+   int i, errors, n, thrd_num, max_req_store;
+   size_t nchars;
+
+   char default_format[FORMAT_STORE];
+   char my_format[] = "host=%20H thrd_num=%0.4n binds_to=%A";
+   char **buffer;
+
+// Display Default Affinity Format
```
Comment (tmh97) [inline tests/5.0/program_control/test_set_get_capture_omp_affinity.c:33]: break get and set into 1 test, capture into another

File: tests/5.0/program_control/test_set_and_get_omp_affinity.c [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   size_t set_nchars;
+   errors = 0;
+    
+   char default_format[FORMAT_STORE];
+   char my_format[] = "host=%20H thrd_num=%0.4n binds_to=%A";
+   char **buffer;
+
+   // Display Default Affinity Format using NULL
+   omp_display_affinity(NULL);
+   printf("\n");
```
Comment (spophale) [inline tests/5.0/program_control/test_set_and_get_omp_affinity.c:34]: Convert to info msg.

Comment (tmh97) [reply tests/5.0/program_control/test_set_and_get_omp_affinity.c]: @spophale Decided to remove lines 33 and 34 instead of converting to info_msg.

**Review by jhdavis8 — APPROVED** [review]

**Comment by tmh97** [comment]
@spophale Review Reminder -- no rush

**Comment by tmh97** [comment]
@spophale is this test good to go?

**Review by spophale — APPROVED** [review]

**Pull request merged**